x-pack/plugins/apm/public/components/app/infra_overview/infra_tabs/empty_prompt.tsx

```diff
@@ -5,38 +5,75 @@
  * 2.0.
  */
 
-import { EuiEmptyPrompt, EuiPageTemplate } from '@elastic/eui';
+import {
+  COLOR_MODES_STANDARD,
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiEmptyPrompt,
+  EuiImage,
+  useEuiTheme,
+} from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
 import React from 'react';
+import noResultsIllustrationDark from '../../../../assets/no_results_dark.svg';
+import noResultsIllustrationLight from '../../../../assets/no_results_light.svg';
 
 export function EmptyPrompt() {
```

Comment:
When the table is rendered the empty prompt from infra plugin is a bit different. Is there any issue to align these two? For example, in the APM view there is no query bar (only a time range and an environment selector)

<img width="1242" alt="image" src="https://user-images.githubusercontent.com/5831975/180757843-0b67fb9d-80b9-4cb5-854d-8f144a6f3e95.png">

Reply:
It's why I left this part out because in our side doesn't make sense the message for the query, we should ask the infra team how to align this
cc @weltenwort

Reply:
cc @boriskirov

Reply:
That's the downside of a shared component, I guess. You could parameterize this so it can be overridden by the consuming UI.